app/src/main/java/com/twilio/video/app/videosdk/RoomManager.kt

```diff
@@ -0,0 +1,899 @@
+package com.twilio.video.app.videosdk
```

Comment:
We should just keep these new classes under the `com.twilio.video.app.ui.room`

This is a similar approach to the io app

https://github.com/google/iosched/blob/master/mobile/src/main/java/com/google/samples/apps/iosched/ui/signin/SignInViewModelDelegate.kt

Reply:
Yep that's fine for now, but I'd argue package by feature is the way to go opposed to packaging by layer. [This quick article](https://hackernoon.com/package-by-features-not-layers-2d076df1964d) explains the benefits. Also, later down the road if we want to modularize the app, each feature package could become its own module and will reduce the dependencies from other modules.